features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -306,17 +319,26 @@ internal class RumFeature(
         vitalExecutorService = NoOpScheduledExecutorService()
         sessionListener = NoOpRumSessionListener()
 
+        cleanupInfoProviders()
+
+        GlobalRumMonitor.unregister(sdkCore)
+    }
+
+    // endregion
+
+    private fun cleanupInfoProviders() {
         if (configuration.collectAccessibility) {
             accessibilityReader.cleanup()
-            accessibilityReader = NoOpAccessibilityReader()
+            accessibilityReader = NoOpInfoProvider()
             accessibilitySnapshotManager = NoOpAccessibilitySnapshotManager()
         }
 
-        GlobalRumMonitor.unregister(sdkCore)
```

Comment:
This line removed on purpose? Don't see it moved somewhere else

Reply:
it moved to line 324